test/tests-harmony.js

```diff
@@ -12779,9 +12710,9 @@ testFail("let default", "Unexpected token (1:4)", {ecmaVersion: 6});
 
 testFail("const default", "Unexpected token (1:6)", {ecmaVersion: 6});
 
-testFail("\"use strict\"; ({ v: eval }) = obj", "Assigning to eval in strict mode (1:20)", {ecmaVersion: 6});
+testFail("\"use strict\"; const { v: eval } = obj", "Binding eval in strict mode (1:25)", {ecmaVersion: 6});
```

Comment:
Wasn't sure if this is what the right course of action was to make these tests pass